src/main/java/com/sigma/cinematicketpro/util/JwtTokenUtils.java

```diff
@@ -0,0 +1,74 @@
+package com.sigma.cinematicketpro.util;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.io.Decoders;
+import io.jsonwebtoken.security.Keys;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Component;
+
+import java.security.Key;
+import java.time.Duration;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+@Component
+public class JwtTokenUtils {
```

Comment:
this is a configuration!

Reply:
It's more appropriate for utility classes to be marked with @Component rather than @Configuration, because classes marked with @Configuration are proxied by Spring differently than @Component which leads to additional redundant functionality applied to bean.

Reply:
discussed